client/src/views/menu/MainMenu.vue

```diff
@@ -690,6 +676,13 @@ onUnmounted(async () => {
   watchWindowWidthCancel();
 });
 
+async function onCategoryMenuChange(menu: WorkspaceMenu): Promise<void> {
+  if (getCurrentRoute.name !== Routes.Workspaces) {
```

Comment:
Correct call for this would be
```suggestion
  if (getCurrentRoute().value.name !== Routes.Workspaces) {
```

but you can use the existing function `currentRouteIs`:
```suggestion
  if (!currentRouteIs(Routes.Workspaces)) {
```